src/workers/webdav/modules/files/domain/RemoteFileContentsRepository.ts

```diff
@@ -7,5 +7,5 @@ export interface RemoteFileContentsRepository {
 
   download(file: WebdavFile): Promise<Readable>;
 
-  upload(size: FileSize, contents: Readable): Promise<WebdavFile['fileId']>;
+  upload(name: string, size: FileSize, contents: Readable): Promise<WebdavFile['fileId']>;
```

Comment:
If the interface changes, when it's used should change also

Reply:
Also, we should avoid passing the name only to add it to the event. If we need to add more info on the event we will have to modify the contract. For the tracking on ruddestack we will need the size and type of the file for example